copper-server-impl/src/main/java/com/pogeyan/cmis/impl/services/CmisObjectService.java

```diff
@@ -2061,6 +2070,9 @@ private static IBaseObject createRelationshipIntern(String repositoryId, String
 				throws CmisInvalidArgumentException, CmisObjectNotFoundException, IllegalArgumentException {
 			MBaseObjectDAO objectMorphiaDAO = DatabaseServiceFactory.getInstance(repositoryId)
 					.getObjectService(repositoryId, MBaseObjectDAO.class);
+			IObjectFlowService objectFlowService = ObjectFlowFactory.createObjectFlowService(repositoryId);
+			invokeObjectFlowServiceBeforeCreate(objectFlowService, repositoryId, folderId, properties, policies,
```

Comment:
the boolean result of this function has to be checked before allowing to go further,